config/config.schema.json

```diff
@@ -664,7 +661,32 @@
           },
           "subscription": {
             "type": "string"
-          }
+          },
+          "istio": {
+            "type": "object",
+            "properties": {
+              "url": {
+                "description": "The download Url for istioctl",
```

Comment:
Are we moving to an explicit link to a release artifact and checking SHASUM like @weinong mentioned?

Reply:
Im not hearing anything yet. i think we can keep the link here for now and update to prefer method in the future.

Reply:
This was in the Teams chat - it should be as easy as using `wget` on the literal release file you need (which the script also does, you can just inspect that) and `md5sum` to check with the published value.